collect_app/src/main/java/org/odk/collect/android/widgets/QuestionWidget.java

```diff
@@ -276,7 +276,7 @@ protected void addHelpTextView(View v) {
 
         // default for helptext
         RelativeLayout.LayoutParams params = new RelativeLayout.LayoutParams(
-                ViewGroup.LayoutParams.WRAP_CONTENT, ViewGroup.LayoutParams.WRAP_CONTENT);
+                ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT);
```

Comment:
For future PRs, make sure to separate out different steps into different commits so the commit message can give a little bit of information about why a certain change is being made!